External/Plugins/CodeFormatter/CodeFormatter.csproj

```diff
@@ -71,26 +71,49 @@
       <HintPath>..\..\..\FlashDevelop\Bin\Debug\Antlr3.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="ProjectManager, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\FlashDevelop\Bin\Debug\Plugins\ProjectManager.dll</HintPath>
+    </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Data" />
     <Reference Include="System.Drawing" />
+    <Reference Include="System.Web" />
     <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
     <Compile Include="AStyle\AStyleInterface.cs" />
+    <Compile Include="Dialogs\HaxeAStyleDialog.cs">
+      <SubType>Form</SubType>
+    </Compile>
+    <Compile Include="Dialogs\HaxeAStyleDialog.Designer.cs">
+      <DependentUpon>HaxeAStyleDialog.cs</DependentUpon>
+    </Compile>
+    <Compile Include="Preferences\HaxeAStyleOption.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="InfoCollector\Utilities.cs" />
     <Compile Include="Handlers\AS3_exLexer.cs" />
     <Compile Include="Handlers\AS3_exParser.cs" />
     <Compile Include="Handlers\MXMLLexer.cs" />
     <Compile Include="PluginMain.cs" />
     <Compile Include="Settings.cs" />
+    <Compile Include="Utilities\HaxeAStyleHelper.cs" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\FlashDevelop\FlashDevelop.csproj">
+      <Project>{EFD07485-9A64-4EEC-94E7-ACBD4DA5CA93}</Project>
+      <Name>FlashDevelop</Name>
+    </ProjectReference>
     <ProjectReference Include="..\..\..\PluginCore\PluginCore.csproj">
       <Project>{61885F70-B4DC-4B44-852D-5D6D03F2A734}</Project>
       <Name>PluginCore</Name>
       <Private>False</Private>
     </ProjectReference>
+    <ProjectReference Include="..\AirProperties\AirProperties.csproj">
+      <Project>{F2CED639-1DA7-489D-9DBB-909F7B590C3D}</Project>
+      <Name>AirProperties</Name>
+    </ProjectReference>
```

Comment:
These references seem unnecessary... I can compile with all of the newly added ones removed (namely, ProjectManager, System.Data, System.Web, System.Xml, FlashDevelop(why was this added?), and AirProperties).